shared/lib/constants.py

```diff
@@ -153,167 +153,198 @@
 
 # Note: These heuristics should be revisited if we change
 # query preprocessing (e.g. stopwords, stemming)
-QUERY_CLASSIFICATION_HEURISTICS: Dict[str, Union[List[str], Dict[
-    str, List[str]]]] = {
-        "Ranking": {
-            "High": [
-                "most",
-                "(?<!bottom to )top",
-                "best",  # leaving here for backwards-compatibility
-                "(?<!lowest to )highest",
-                "high",
-                "largest",
-                "biggest",
-                "greatest",
-                "strongest",
-                "richest",
-                "sickest",
-                "illest",
-                "oldest",
-                "major",  # as in 'major storms'
-                "descending",
-                "top to bottom",
-                "highest to lowest",
-            ],
-            "Low": [
-                "least",
-                "(?<!top to )bottom",
-                "worst",  # leaving here for backwards-compatibility
-                "(?<!highest to )lowest",
-                "low",
-                "smallest",
-                "weakest",
-                "youngest",
-                "poorest",
-                "healthiest",
-                "ascending",
-                "bottom to top",
-                "lowest to highest",
-            ],
-            "Best": ["best",],
-            "Worst": ["worst",],
-            "Extreme": ["extremes?", "impact"]
-        },
-        "Comparison": [
-            "compare(s|d)?",
-            "comparison",
-            "(is|has|have)( a| the)? \w+er",
-            # WARNING: These will conflate with Correlation
-            "vs",
-            "versus",
+QUERY_CLASSIFICATION_HEURISTICS: Dict[str, Union[List[str], Dict[str, List[str]]]] = {
```

Comment:
Is there anything actually changing in this dict, or are these just formatting diffs?